File: android/src/main/java/com/piccmaq/flutter_paypal_native/models/CheckoutConfigStore.java
package com.piccmaq.flutter_paypal_native.models;

import com.paypal.checkout.config.Environment;
import com.paypal.checkout.createorder.CurrencyCode;
import com.paypal.checkout.createorder.UserAction;

public class CheckoutConfigStore {

   public String clientId ;
    public  Environment payPalEnvironment ;
    public  String returnUrl;
    public CurrencyCode currency ;
    public UserAction userAction ;

    public CheckoutConfigStore(
            String clientId,
            Environment payPalEnvironment,
            String returnUrl,
            CurrencyCode currency,
            UserAction userAction
    ) {
        this.payPalEnvironment = payPalEnvironment;
        this.clientId = clientId;
        this.returnUrl = returnUrl;
        this.currency = currency;
        this.userAction = userAction;
    }


}

File: ios/Classes/models/CheckoutConfigStore.swift
import PayPalCheckout

public class CheckoutConfigStore {
    let clientId: String
    let payPalEnvironment: Environment
    let returnUrl: String
    let currency: CurrencyCode
    let userAction: UserAction

    public init(clientId: String, payPalEnvironment: Environment, returnUrl: String, currency: CurrencyCode, userAction: UserAction) {
        self.clientId = clientId
        self.payPalEnvironment = payPalEnvironment
        self.returnUrl = returnUrl
        self.currency = currency
        self.userAction = userAction
    }
}